sequential/seq2pat.py

```diff
@@ -1,19 +1,22 @@
 # -*- coding: utf-8 -*-
 # SPDX-License-Identifier: GPL-2.0
 
+import gc
 from typing import NamedTuple, List, Dict, NoReturn
-from sequential.utils import Num, check_true, check_false, get_max_column_size, \
-    get_min_value, get_max_value, sort_pattern, item_map,\
-    string_to_int, int_to_string, check_sequence_feature_same_length
+
 from sequential.backend import seq_to_pat as stp
+from sequential.utils import Num, check_true, get_max_column_size, \
+    get_min_value, get_max_value, sort_pattern, item_map, \
+    string_to_int, int_to_string, check_sequence_feature_same_length, \
+    validate_attribute_values, validate_sequences
+from sequential._version import __version__
 
-__version__ = "1.2.2"
+__version__ = __version__
```

Comment:
I'm assuming you don't need this line?

Reply:
Yes, the version information is not used in the script though. It could be ok to drop.